src/select/SimpleSelect.js

```diff
@@ -578,68 +564,62 @@ class SimpleSelect extends Component {
   }
 
   renderArrow(isOpen) {
-    const ArrowComponent = this.props.arrowComponent
+    const { arrowComponent } = this.props
+    const ArrowComponent = arrowComponent
     const { arrowRenderer } = this.props
 
-    return (<ArrowComponent
-      arrowRenderer={arrowRenderer}
-      onArrowClick={this.onArrowClick}
-      onArrowTouchEnd={this.onArrowTouchEnd}
-      isOpen={isOpen}
-    />)
+    return (
+      <ArrowComponent
+        arrowRenderer={arrowRenderer}
+        onArrowClick={this.onArrowClick}
+        onArrowTouchEnd={this.onArrowTouchEnd}
+        isOpen={isOpen}
+      />
+    )
   }
 
   renderClear() {
-    const ClearComponent = this.props.clearComponent
-    const {
-      clearable,
-      clearRenderer,
-      disabled,
-      value
-    } = this.props
+    const { clearComponent } = this.props
+    const ClearComponent = clearComponent
+    const { clearable, clearRenderer, disabled, value } = this.props
```

Comment:
super nit: could combine lines 582 and 584

Reply:
Fixed!